ibmcloud/SECURE_EXECUTION.md

```diff
@@ -0,0 +1,124 @@
+
+When you obtain the host key document, please copy the downloaded host key document `HKD-<type>-<serial>.crt` to the worker node host keys directory, the directory can be any, if you use `/root/hostkeys/` as the host keys directory, you can use follow commands:
+```bash
+ssh root@ip-of-peer-pod-worker-node mkdir /root/hostkeys/
+scp HKD-<type>-<serial>.crt root@ip-of-peer-pod-worker-node:/root/hostkeys/
+```
+
+> **NOTE**: You can download multiple different host keys and prepare only one SE enabled image for different host systems.
+>
+> eg. If you use `/root/hostkeys/` as the hosy keys directory and you want to deploy the SE enabled image on two host systems which the machine type are `8562` the machine serial number are `1234567` and `7654321` then the hosy keys directory tree looks like as follow:
+> ```bash
+> ssh root@ip-of-peer-pod-worker-node tree /root/hostkeys/
+> ...
+> /root/hostkeys/
+> ├── HKD-8562-1234567.crt
+> └── HKD-8562-7654321.crt
+> 
+> 0 directories, 2 files
+> ```
+
+## Build the SE enabled qcow2 image on worker node
+
+Still use the [build script](./image/build.sh) for IBM cloud image. To build SE enabled image, please set `SE_BOOT` as 1, and set `HOST_KEYS_DIR` with the host key documents directory (eg. `/root/hostkeys/`) and set `IMAGE_NAME` with a `se-` prefix (eg. `se-podvm-s390x`) then run follow commands:
+```
+ssh root@ip-of-peer-pod-worker-node
+cd /root/cloud-api-adaptor/ibmcloud/image
+CLOUD_PROVIDER=ibmcloud SE_BOOT=1 HOST_KEYS_DIR=/root/hostkeys/ IMAGE_NAME=se-podvm-s390x make build
+```
+> **Notes** 
+> - The [build script](./image/build.sh) is following the steps in [the document](https://www.ibm.com/docs/en/linux-on-systems?topic=execution-workload-owner-tasks)
+> - The `IMAGE_NAME` is must end with `-s390x` or `-amd64`
+
+## Upload the SE enabled qcow2 image and creat one SE enabled custom image for IBM Cloud VPC
+
+Please confirm the IBM Cloud API key, COS instance name and COS bucket name, VPC name, subnet name, region and zone when you set up the demo environment, set `SE_BOOT` as 1 and set `IMAGE_NAME` with a `se-` prefix (eg. `se-podvm-s390x`) then run follow commands:
+```
+export IBMCLOUD_API_KEY=<your_api_key>
+export IBMCLOUD_API_ENDPOINT="https://cloud.ibm.com"
+export IBMCLOUD_COS_REGION=<the region your IKS cluster and zVSI is running on>
+export IBMCLOUD_COS_SERVICE_INSTANCE=<your_cos_service_instance_name>
+export IBMCLOUD_COS_BUCKET=<your_cos_bucket_name>
+export IBMCLOUD_COS_SERVICE_ENDPOINT="https://s3.${IBMCLOUD_COS_REGION}.cloud-object-storage.appdomain.cloud"
+export IBMCLOUD_VPC_NAME="<your_VPC_name>"
+export IBMCLOUD_VPC_SUBNET_NAME="<your_subnet_name>"
+export IBMCLOUD_VPC_REGION="<the_region_name>"
+export IBMCLOUD_VPC_ZONE="<the_zone_name>"
+
+CLOUD_PROVIDER=ibmcloud SE_BOOT=1 IMAGE_NAME=se-podvm-s390x make push
```

Comment:
nit: With the replacement for push now also redoing the build, could we instead combine the build steps above and just do the push?

Another thing is we could instead export `IMAGE_NAME` so that we keep consistent with other docs, and we decrease the chance of the user making a mistake between build, push and verify. 
So instead adding: 
```
export IMAGE_NAME=se-<image-name>-<arch>
```
to the list of exported variables above. And that way it should remain the same for each run.

Reply:
updated.
use exported variables.